CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.4)
 project(cmake_example)
 
+# EXAMPLE_VERSION_INFO will be defined by setup.py
```

Comment:
Why the rename of `VERSION_INFO`? This makes the `CMakeBuild` quite a bit less easy to "plug in".

Reply:
Because a) you shouldn't be passing raw flags though to the compilee, b) raw flags unquote differently on Windows, and c) this matches python_example.

Reply:
(But I could make the define exactly the same name, maybe that's what you meant?)

Reply:
I originally left them different so that users could tell which was which; if they were the same they might mix a CMake define with a C++ define.

Reply:
I see, yes. I was thinking of users (*cough cough me cough*) copying this class to their own `setup.py`. But I guess this is not a huge change to make. Could potentially be documented, though, that this should be changed when using in your own project.